.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,81 @@
+    validations:
+      required: false
+
+  - type: checkboxes
+    id: contribution
+    attributes:
+      label: Contribution
+      description: Are you willing to help implement this feature?
+      options:
+        - label: I would like to implement this feature
+        - label: I can help with documentation
+        - label: I can help with testing
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)